src/main/java/fiuba/tpp/reactorapp/model/dto/SearchByAdsorbateDTO.java

```diff
@@ -0,0 +1,52 @@
+package fiuba.tpp.reactorapp.model.dto;
+
+import fiuba.tpp.reactorapp.entities.Adsorbate;
+import fiuba.tpp.reactorapp.entities.Process;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SearchByAdsorbateDTO {
+
+    private List<Process> processes;
+
+    private List<Adsorbate> removeAdsorbates;
+
+    private float maxQmax;
+
+    public SearchByAdsorbateDTO(Process process) {
+        this.processes = new ArrayList<>();
+        this.removeAdsorbates = new ArrayList<>();
+        processes.add(process);
+        this.removeAdsorbates.add(process.getAdsorbate());
+        maxQmax = process.getQmax();
```

Comment:
Te faltaron un par de `this` acá, se que es lo mismo pero por consistencia o usaría this en todo o en ninguna de las lineas